_specification-v2/manifest.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Origami.json Manifest Specification
+description: A specification which describes the required structure of an Origami.json manifest file.
+cta: Read the manifest spec
+
+# Redirect from legacy URLs
+redirect_from:
+  - /docs/syntax/origamijson/
+
+# Navigation config
+nav_display: true
+nav_label: Manifest
+nav_heading_selector: h1, h2, h3
+nav_order: 25
+---
+
+# {{page.title}}
+
+`origami.json` is a
+<a href="https://www.json.org/"
+class="o-typography-link--external">
+	<abbr title="JavaScript Object Notation">
+		JSON
+	</abbr>
+</a>
+file responsible for describing various aspects of an Origami project.
+
+## Properties
+
+### origami
+
+<table class="o-manifest__table o-table o-table--compact o-table--row-headings o-table--vertical-lines o-table--horizontal-lines" data-o-component="o-table">
+	<tr>
+		<th scope="row" role="rowheader">Type</th>
+		<td><code>String</code></td>
+	</tr>
+	<tr>
+		<th scope="row" role="rowheader">Required</th>
+		<td><code>true</code></td>
+	</tr>
+</table>
+
+The version of Origami to which the project conforms.
+
+**Must** be set to `"2.0"`.
+
+<pre><code class="o-syntax-highlight--json">{
+	"origami": "2.0"
+}</code></pre>
+
+### type
+
+<table class="o-manifest__table o-table o-table--compact o-table--row-headings o-table--vertical-lines o-table--horizontal-lines" data-o-component="o-table">
+	<tr>
+		<th scope="row" role="rowheader">Type</th>
+		<td><code>String</code> or <code>null</code></td>
+	</tr>
+	<tr>
+		<th scope="row" role="rowheader">Required</th>
+		<td><code>true</code></td>
+	</tr>
+</table>
+
+Defines the type of Origami project that the manifest belongs to. **Must** be
+set to a string:
```

Comment:
we need to mention it an be set to `null`, which is also in the list of possible options
For the code suggestion, I copied the text from the `status` part of this spec which uses "must be one of:"
```suggestion
Defines the type of Origami project that the manifest belongs to. **Must** be one of:
```

Reply:
oh yes, this was something I wanted to talk about.
I'm wondering if we should say something like:

> **must** be set to a string, .e.g.:

and then remove `null` as an option. `type` then becomes an entry point for extensions. the component spec augments the spec by saying "when the `type` is "component" then all of this applies:" and you opt into this set of rules by setting the type to that

Reply:
I like that idea a lot.

I checked what sets it to null, and the list is:
- change-api-action -- github action
- origami-apply-labels -- github action
- origami-labels -- github action
- origami-percy -- github action
- origami-project-board-action -- github action
- origami-version -- github action
- polyfill-library-node -- npm libary
- useragent_parser -- npm library

Reply:
```suggestion
Defines the type of Origami project that the manifest belongs to. **Must** be set to a string, e.g.:
```

Reply:
nice work! I've added a description of `"action"` to this list, we can update those to use that. and maybe even we write a spec for our github actions some time:)